.github/workflows/action.yml

```
@@ -0,0 +1,38 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Github Actions for Elephant Vending Machine - Front-end
```

```suggestion
name: build
```

This name should describe what is happening in the workflow. I think if we make it `build` then the badge on the repo will show "Build: Passing" so it's clear what the status of the action is.